server/server.go

```diff
@@ -508,12 +521,14 @@ func (s *Server) LoopContext() context.Context {
 
 func (s *Server) startServerLoop(ctx context.Context) {
 	s.serverLoopCtx, s.serverLoopCancel = context.WithCancel(ctx)
-	s.serverLoopWg.Add(5)
+	s.serverLoopWg.Add(4)
 	go s.leaderLoop()
 	go s.etcdLeaderLoop()
 	go s.serverMetricsLoop()
-	go s.tsoAllocatorLoop()
 	go s.encryptionKeyManagerLoop()
+	// TODO: after support redirect TSO request we need to skip it.
```

Comment:
then no need change this code for now, or you can add a TSO request redirected flag, set it to the default value false, and put the two lines below to the if (! tsoRequestRedirected) block. This code now looks not good.

Reply:
change it back